src/main/java/com/ft/content/model/ContentPackage.java

```diff
@@ -0,0 +1,52 @@
+package com.ft.content.model;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ContentPackage {
+
+    private final String description;
+
+    private final String listLink;
```

Comment:
How does one value would look like? If just an uuid, then I would go with the `id` naming which is consistent with our existing model (e.g. Brand)